tests/get_packages.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+import tomli
+
+platform = sys.platform
+distributions = sys.argv[1:]
+if not distributions:
+    distributions = os.listdir("stubs")
+
+metadata_mapping = {
+    "linux": "apt_dependencies",
+    # TODO: support others:
+    # "darwin": "brew_dependencies",
+    # "win32": "choco_dependencies",
```

Comment:
I like that you've done this in an extensible way, but this doesn't need to be a TODO comment for now; we don't have any stubs that currently need installations using `brew` or `choco`

```suggestion
    # We could add others here if we run into stubs that need it:
    # "darwin": "brew_dependencies",
    # "win32": "choco_dependencies",
```